packages/web-components/fast-components/src/toolbar/toolbar.styles.ts

```diff
@@ -65,7 +64,6 @@ export const ToolbarStyles: (context, definition) => ElementStyles = (
             height: 16px;
         }
     `.withBehaviors(
-        neutralFillCardRestBehavior,
```

Comment:
Just checking that this was intended.

Reply:
yea - these behaviors are no longer needed and design tokens should be used in-place.